autofs-5.1.4 - fix rpm spec install permissions on auto.net and auto.smb

From: Ian Kent <raven@themaw.net>

Commit 0da3f5695d removes execute permissions from the install of
auto.net and auto.smb.

Signed-off-by: Ian Kent <raven@themaw.net>
---
 CHANGELOG   |    1 +
 autofs.spec |    4 ++--
 2 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/CHANGELOG b/CHANGELOG
index 596590a8..a5b79dd6 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -20,6 +20,7 @@ xx/xx/2018 autofs-5.1.5
 - add conditional inclusion of fedfs binaries.
 - add an example fedfs master map entry to the installed master map.
 - improve hostname lookup error logging.
+- fix rpm spec install premissions on auto.net and auto.smb.
 
 19/12/2017 autofs-5.1.4
 - fix spec file url.
diff --git a/autofs.spec b/autofs.spec
index 2cb09ef0..ac3b0a3f 100644
--- a/autofs.spec
+++ b/autofs.spec
@@ -131,8 +131,8 @@ install -m 644 redhat/autofs.sysconfig $RPM_BUILD_ROOT/etc/sysconfig/autofs
 
 install -m 644 samples/auto.master $RPM_BUILD_ROOT/etc/auto.master
 install -m 644 samples/auto.misc $RPM_BUILD_ROOT/etc/auto.misc
-install -m 644 samples/auto.net $RPM_BUILD_ROOT/etc/auto.net
-install -m 644 samples/auto.smb $RPM_BUILD_ROOT/etc/auto.smb
+install -m 755 samples/auto.net $RPM_BUILD_ROOT/etc/auto.net
+install -m 755 samples/auto.smb $RPM_BUILD_ROOT/etc/auto.smb
 install -m 600 samples/autofs_ldap_auth.conf $RPM_BUILD_ROOT/etc/autofs_ldap_auth.conf
 
 %clean
